src/signals/incident/components/form/MapSelectors/Asset/Selector/LegendPanel/LegendPanel.test.tsx

```diff
@@ -1,15 +1,16 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2021 Gemeente Amsterdam
 import { render, screen } from '@testing-library/react'
+import 'jest-styled-components'
+
```

Comment:
what is this import for?

Reply:
This wil export the `toHaveStyleRule` assertion as used in the added test cases in this suite